.travis.yml

```diff
@@ -94,7 +98,6 @@ deploy:
     distributions: sdist
     username: __token__
     skip_existing: true
-    skip_cleanup: true
```

Comment:
Removed because deprecated and now default behaviour, right ?

Reply:
yes exactly :)